examples/openassetio/openassetio.otio

```diff
@@ -0,0 +1,50 @@
+{
+    "tracks": {
+        "OTIO_SCHEMA": "Stack.1",
+        "children": [
+            {
+                "name": "video",
+                "OTIO_SCHEMA": "Track.1",
+                "children": [
+                    {
+                        "name": "rv_asset1",
+                        "OTIO_SCHEMA": "Clip.1",
+                        "source_range": {
+                            "start_time": {
+                                "OTIO_SCHEMA": "RationalTime.1",
+                                "value": 1.0,
+                                "rate": 24.0
+                            },
+                            "duration": {
+                                "OTIO_SCHEMA": "RationalTime.1",
+                                "value": 49.0,
+                                "rate": 24.0
+                            },
+                            "OTIO_SCHEMA": "TimeRange.1"
+                        },
+                        "media_reference": {
+                            "OTIO_SCHEMA": "ExternalReference.1",
+                            "available_range": {
+                                "start_time": {
+                                    "OTIO_SCHEMA": "RationalTime.1",
+                                    "value": 1.0,
+                                    "rate": 24.0
+                                },
+                                "duration": {
+                                    "OTIO_SCHEMA": "RationalTime.1",
+                                    "value": 49.0,
+                                    "rate": 24.0
+                                },
+                                "OTIO_SCHEMA": "TimeRange.1"
+                            },
+                            "target_url": "rv_asset1"
```

Comment:
The `bal:///` string needs to be here instead, as that's an [implementation detail of BAL itself](https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/blob/bdc28e9805923cc510ca237f62b9b49acba3b882/plugin/openassetio_manager_bal/BasicAssetLibraryInterface.py#L109).

Reply:
I was wondering about that actually.  Good to know! Will make the changes.